more_itertools/more.py

```diff
@@ -128,6 +128,14 @@ def chunked(iterable, n):
         >>> list(chunked([1, 2, 3, 4, 5, 6, 7, 8], 3))
         [[1, 2, 3], [4, 5, 6], [7, 8]]
 
+    If this is the case and *strict* is True, then :func:`chunked` will
```

Comment:
Maybe change this to:
"If `strict=True`, `chunked` raises a `ValueError` if `iterable` is not divisible by `n`."